dashboard/src/shared/types.ts

```diff
@@ -479,6 +480,6 @@ export interface IBasicFormSliderParam extends IBasicFormParam {
 }
 
 export interface CustomInstalledPackageDetail extends InstalledPackageDetail {
-  manifest: any;
+  apiResourceRefs: ResourceRef[];
```

Comment:
I don't think it really belongs in here but in the `InstalledPackageDetail` intrerface, I think?

Reply:
Ah - ok. I'll move it and delete the CustomInstalledPackageDetail then.

Reply:
Hmm, so just checked: we can't put it on `InstalledPackageDetail` given that `InstalledPackageDetail` is generated from the proto. I'd added it here simply because the manifest was there (and this field replaced the manifest). I think we'll need to instead pass it as a separate value, with the InstalledPackageDetail.

Reply:
Ahh, I thought the `InstalledPackageDetail` (proto) already has this `ResourceRef` field! Got it then, makes sense to keep it here, temporarily.
Thanks for the clarification!

Reply:
Cool. Just to be clear though, the `InstalledPackageDetail` proto won't include the resource refs, since it's a separate request. We were originally thinking about just including it on the detail, but after discussion, went for a separate call.